CMake/deps/Boost.cmake

```diff
@@ -1,6 +1,11 @@
 set(Boost_USE_STATIC_LIBS ON)
 set(Boost_USE_STATIC_RUNTIME ON)
 
-find_package(Boost 1.75 COMPONENTS filesystem log_setup log thread system REQUIRED)
-
-target_link_libraries(clio PUBLIC ${Boost_LIBRARIES})
+find_package(Boost 1.82 REQUIRED
```

Comment:
all the boost modules are linked now

Reply:
that's a shame, i'd imagine with `find_package` approach and the explicit libraries linked using `target_link_libraries` we should only get what we asked for. i have noticed a bunch of abseil libs linked. maybe we can fix that later if we find a way.